eng/scripts/Generate-Patch.ps1

```diff
@@ -39,37 +41,39 @@ Write-Information "ArtifactName is: $ArtifactName"
 Write-Information "ReleaseVersion is: $ReleaseVersion"
 Write-Information "ServiceDirectoryName is: $ServiceDirectoryName"
 
-$MainRemoteUrl = 'https://github.com/Azure/azure-sdk-for-java.git'
+$MainRemoteUrl = 'https://github.com/azure/azure-sdk-for-java.git'
 $RepoRoot = Resolve-Path "${PSScriptRoot}..\..\.."
 $EngDir = Join-Path $RepoRoot "eng"
 $EngCommonScriptsDir = Join-Path $EngDir "common" "scripts"
 $SdkDirPath = Join-Path $RepoRoot "sdk"
 $ServiceDirPath = Join-Path $SdkDirPath $ServiceDirectoryName
 $ArtifactDirPath = Join-Path $ServiceDirPath $ArtifactName
-$ArtifactPomFile = Join-Path $ArtifactDirPath "pom.xml"
-$ReleaseTag = -join($ArtifactName, "_", $ReleaseVersion)
 $GroupId = "com.azure"
 
 
 TestPathThrow -Path $RepoRoot
 
 . (Join-Path $EngCommonScriptsDir common.ps1)
+. (Join-Path $EngCommonScriptsDir SemVer.ps1)
```

Comment:
This should already be imported in common.ps1 so it shouldn't need to be explicitly added here.